### ISSUE TITLE
Header Error

### ISSUE THREAD

**Issue by zek** [mvayngrib/react-native-randombytes#31]
~~~
In file included from /Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:9:
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.h:12:21: error: expected "FILENAME" or <FILENAME>
#elif __has_include(“RCTBridgeModule.h”)
                    ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.h:12:21: error: invalid token at start of a preprocessor expression
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.h:15:9: error: expected "FILENAME" or <FILENAME>
#import “React/RCTBridgeModule.h” // Required when used as a Pod in a Swift project
        ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.h:19:37: error: no type or protocol named 'RCTBridgeModule'
@interface RNRandomBytes : NSObject<RCTBridgeModule>
                                    ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:12:21: error: expected "FILENAME" or <FILENAME>
#elif __has_include(“RCTBridgeModule.h”)
                    ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:12:21: error: invalid token at start of a preprocessor expression
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:15:9: error: expected "FILENAME" or <FILENAME>
#import “React/RCTBridgeModule.h” // Required when used as a Pod in a Swift project
        ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:22:1: error: expected function body after function declarator
@synthesize bridge = _bridge;
^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:24:19: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
RCT_EXPORT_METHOD(randomBytes:(NSUInteger)length
                  ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:24:30: error: expected ')'
RCT_EXPORT_METHOD(randomBytes:(NSUInteger)length
                             ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:24:18: note: to match this '('
RCT_EXPORT_METHOD(randomBytes:(NSUInteger)length
                 ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:24:1: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
RCT_EXPORT_METHOD(randomBytes:(NSUInteger)length
^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:33:5: warning: ignoring return value of function declared with 'warn_unused_result' attribute [-Wunused-result]
    SecRandomCopyBytes(kSecRandomDefault, length, [bytes mutableBytes]);
    ^~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:27:5: error: implicit declaration of function 'callback' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    callback(@[[NSNull null], [self randomBytes:length]]);
    ^
/Users/zack/Desktop/DERS/react/test/node_modules/react-native-randombytes/RNRandomBytes.m:27:32: error: use of undeclared identifier 'self'
    callback(@[[NSNull null], [self randomBytes:length]]);
                               ^
3 warnings and 11 errors generated.



** BUILD FAILED **
~~~

**Comment by m-tymchyk** [comment]
I have the same issue.

**Comment by qispark** [comment]
Try my pull request - replacing the curly quotes seems to make it compile correctly.

**Comment by mvayngrib** [comment]
closing, fixed in #35
